code/modules/surgery/bodyparts/head.dm

```diff
@@ -247,6 +247,7 @@
 				if(sprite_accessory)
 					//Create the overlay
 					facial_overlay = mutable_appearance(sprite_accessory.icon, sprite_accessory.icon_state, -HAIR_LAYER)
+					facial_overlay.overlays += emissive_blocker(facial_overlay.icon, facial_overlay.icon_state, alpha = facial_overlay.alpha)
```

Comment:
```suggestion
					facial_overlay.overlays += emissive_blocker(facial_overlay.icon, facial_overlay.icon_state, alpha = hair_alpha)
```

Reply:
I wonder if there's actually a facial_hair_alpha, now that I think about it

Reply:
I didn't make one because I didn't forsee it to even be possible for them to have seperate alphas.

Reply:
Okay, good, this means I can just commit this without fearing any issue, good.